fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBDatabase.java

```diff
@@ -912,6 +913,7 @@ protected RuntimeException mapAsyncToSyncException(@Nonnull Throwable ex) {
             } catch (TimeoutException ex) {
                 if (timer != null) {
                     timer.recordTimeout(event, startTime);
+                    throw asyncToSyncExceptionMapper.apply(new LoggableTimeoutException(ex, LogMessageKeys.TIME_LIMIT.toString(), timeout.getLeft(), LogMessageKeys.TIME_UNIT.toString(), timeout.getRight()), event);
```

Comment:
No need to fix, but the `.toString()` is necessary for the `LogMessageKeys`.  The logging framework automatically does this.

Reply:
Interesting that things were failing without that.